app/components/Modal/BorrowModal.jsx

```diff
@@ -227,6 +227,35 @@ class BorrowModalContent extends React.Component {
         );
     }
 
+    _maximizeDebt() {
+        let backing_balance = !this.props.backing_balance
+            ? {balance: 0, id: null}
+            : this.props.backing_balance.toJS();
+        let backingBalance = backing_balance.id
+            ? ChainStore.getObject(backing_balance.id)
+            : null;
+        window.s = backingBalance;
```

Comment:
Why assign this to the window?

Reply:
That's my debugging technique, accidentally left in :disappointed: